server/graphql/v1/mutations/collectives.js

```diff
@@ -200,6 +200,9 @@ export async function createCollectiveFromGithub(_, args, req) {
     collectiveData.LastEditedByUserId = user.id;
     collective = await models.Collective.create(collectiveData);
     const host = await models.Collective.findByPk(defaultHostCollective('opensource').CollectiveId);
+    if (user.email.match(/.*test.*host@opencollective.com$/)) {
```

Comment:
I suppose that this was added for `test/cypress/integration/04-collective.test.js`? Even if it's not for production, adding you as an admin of the host when you create a collective under this host is a surprising side-effect. It's usually beneficial to keep the differences between prod & non-prod env as tiny as possible.

For this one, I would suggest using the default test account (`defaultTestUserEmail`) in Cypress. This account is already an admin of BrusselsTogetherASBL, as you can see in `test/cypress/integration/29-host.dashboard.test.js`.

Reply:
@Betree : Please see my comment at, https://github.com/opencollective/opencollective-frontend/pull/5922#discussion_r620624265